components/organisms/Footer.js

```diff
@@ -0,0 +1,99 @@
+import { format } from "prettier";
+import PropTypes from "prop-types";
+import { ActionButton } from "../atoms/ActionButton";
+import Image from "next/image";
+
+/**
+ * footer element for all pages
+ */
+
+export function Footer(props) {
+  return (
+    <div className="w-full mt-4">
+      <div className="w-full h-48 flex flex-col md:flex-row  bg-footer-blue bg-footer-parliament-image bg-no-repeat bg-right-bottom text-white">
+        <div className="w-full flex flex-col mt-2 mb-2 layout-container pt-4">
+          <div className="w-full flex ">
+            <div className="ml-2 md:ml-0 mr-2 md:mr-0 flex md:items-start mb-2">
+              <p className="mb-4 mt-2 text-sm">
+                {props.reportDescription}
+                <br></br> {props.reportDescriptionContinue}
+              </p>
+            </div>
+          </div>
+          <div className="w-full flex ">
+            <div className="w-1/2  md:w-2/3 ml-2 md:ml-0 mr-2 md:mr-0 flex justify-between items-center md:items-start mb-2">
+              <ActionButton
+                text={props.reportBtnText}
+                secondary
+                dataCyButton={"report-problem-button"}
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="w-full h-auto flex flex-col md:flex-row  items-center bg-footer-white mt-6  justify-between layout-container  ">
+        <div className="w-full flex flex-col md:flex-row  justify-between ">
+          {props.links.map((value, index) => {
+            return (
+              <a
+                className="text-canada-ca-link-colour p-2 md:pt-4 md:pr-4  md:pb-4   hover:underline"
+                key={index}
+                href={value.link}
+                style={{
+                  height: "33%",
+                }}
+              >
+                {value.linkText}
+              </a>
+            );
```

Comment:
We should avoid inline styles for this.

Whenever you see a list of links, like in the footer sections, you should put them in `<ul>` elements. Set the `li`s to inline block and then you don't need inline styling hacks.